static/js/dashboard_routes.js

```diff
@@ -0,0 +1,99 @@
+import App from './containers/App';
+
+const errorLoading = error => {
+  throw new Error(`Dynamic page loading failed: ${error}`);
+};
+
+const loadRoute = cb => (
+  module => cb(null, module.default)
+);
+
+const username = SETTINGS.user ? SETTINGS.user.username : '';
+
+export const routes = {
+  path: "/",
+  component: App,
+  childRoutes: [
+    {
+      path: 'dashboard',
+      getComponent(location, cb) {
+        import('./containers/DashboardPage')
+          .then(loadRoute(cb, false))
```

Comment:
It looks like `loadRoute` only takes one argument, can we remove the `, false` here and below?